salt/utils/win_functions.py

```diff
@@ -210,3 +212,72 @@ def escape_meta_chars(m):
         return meta_map[char]
 
     return meta_re.sub(escape_meta_chars, arg)
+
+
+def broadcast_setting_change(message='Environment'):
+    '''
+    Send a WM_SETTINGCHANGE Broadcast to all Windows
+
+    Args:
+
+        message (str):
+            A string value representing the portion of the system that has been
+            updated and needs to be refreshed. Default is ``Environment``. These
+            are some common values:
+
+            - "Environment" : to effect a change in the environment variables
+            - "intl" : to effect a change in locale settings
+            - "Policy" : to effect a change in Group Policy Settings
+            - a leaf node in the registry
+            - the name of a section in the ``Win.ini`` file
+
+            See lParam within msdn docs for
+            `WM_SETTINGCHANGE <https://msdn.microsoft.com/en-us/library/ms725497%28VS.85%29.aspx>`
+            for more information on Broadcasting Messages.
+
+            See GWL_WNDPROC within msdn docs for
+            `SetWindowLong <https://msdn.microsoft.com/en-us/library/windows/desktop/ms633591(v=vs.85).aspx>`
+            for information on how to retrieve those messages.
+
+    .. note::
+        This will only affect new processes that aren't launched by services. To
+        apply changes to the path or registry to services, the host must be
+        restarted. The ``salt-minion``, if running as a service, will not see
+        changes to the environment until the system is restarted. Services
+        inherit their environment from ``services.exe`` which does not respond
+        to messaging events. See
+        `MSDN Documentation <https://support.microsoft.com/en-us/help/821761/changes-that-you-make-to-environment-variables-do-not-affect-services>`
```

Comment:
This link syntax doesn't look quite right. I think you need a `_`. http://docutils.sourceforge.net/docs/user/rst/quickref.html#external-hyperlink-targets

This goes for all of the other places you have links in the docs for this function, and in the `rehash` note above.

Reply:
That's if you're linking to the links later on. I was following the sphinx documentation.
http://www.sphinx-doc.org/en/stable/rest.html#external-links

Does that not work for inline docs?

Reply:
You still need the final `_` to create the external link: 

```
`Link text <http://example.com/>`_
```